### ISSUE TITLE
Solr with basic auth seems that does not work

### ISSUE THREAD

**Issue by stavros-k** [eikek/docspell#1769]
Setting in config the solr URL like: `http://user:pass@solr:8983/solr/docspell` results in 401 Unauthorized when joex tries to commit data. Even doing `/query` results in 401 Unauthorized. 

Copying the exact same URL and doing `curl http://user:pass@solr:8983/solr/docspell/query` works fine.
Which hints that it might be docspell does not support that?

Let me know if you have any ideas what to try next. 

---

Using bitnami's solr.
Deployment is on k3s single node cluster

**Comment by eikek** [comment]
Yes, it is not supported atm. The encoded url should be used as given though. But I think the underlying http library doesn't make use  of it by default. Curl produces the `Authorization: Basic …` header with the values taken from the url. The `user:pass@` thingy is a feature of the client, but is discouraged to use (some browsers also don't support it anymore).

**Comment by stavros-k** [comment]
Thanks for the quick reply. I guess this could be turned to a feature request?

**Comment by eikek** [comment]
Yes sure, we can leave this issue open.
